private/testeth/testtoken/token.go

```diff
@@ -0,0 +1,788 @@
+// Code generated - DO NOT EDIT.
```

Comment:
A `go:generate` annotation (in `gen.go`) with the command which were used can be helpful to repeat the generation later.

Reply:
that's the plan